### PR TITLE
{tools}[dummy/] git-lfs v2.7.1

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/545d5ba0c3423b73fe7559846dce1c5e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3131.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/ff55b72ba93b5f2ad89c95747133368f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2603.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/375ebdf96d1ec508dba459d35ce15b60 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2047.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e6dc68ea62ab07ff82d872ace3a805bf for a full test report.

File: easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+homepage = 'https://git-lfs.github.com/'
+description = """Git Large File Storage (LFS) replaces large files such as audio samples, videos, 
+ datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server 
+ like GitHub.com or GitHub Enterprise."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/git-lfs/git-lfs/releases/download/v%(version)s/']
+sources = ['%(name)s-linux-amd64-v%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb]: @vanzod Can we build this from source instead?

Maybe it's also worth having a simple sanity check command (certainly when installing it as a binary)...

File: easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+checksums = ['7be85238cbbb957ab25de52b60279d40ba40d3faa72eeb2cb9fa77d6d92381e5']
+
+sanity_check_paths = {
+    'files': ["git-lfs"],
+    'dirs': [""]
```
Comment (boegel) [inline easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb]: @vanzod Please drop the `""` here

File: easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+# add the installation dir to PATH
+modextrapaths = {
+    'PATH': '',
```
Comment (boegel) [inline easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb]: @vanzod single line?

File: easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+# remove exec permission for the provided installation script
+postinstallcmds = ["chmod og-x %(installdir)s/install.sh"]
```
Comment (boegel) [inline easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb]: Why do we still need the install script? Just remove it?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/a04a2c3b761ecb52c018cb70d254b51a for a full test report.

**Comment by vanzod** [comment]
@boegel I got rid of the old tests since now I am building from source instead of using the binary. Please test it again.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/787699fee933e9abb116330c742cbd34 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/51ede3cdc1dcd13725498cd38f26587a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2002.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9a62f4e40322c074a357f898c7257411 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**